app/assets/javascripts/angular/controller/user-show-controller.js

```diff
@@ -7,6 +7,10 @@ app.controller('UserCtrl', ['$scope', '$http',
   //   console.log(data);
   //   $scope.user = data.user;
   // })
-
+  
+    $http.get('pins.json').success(function(data){
```

Comment:
This is a partial for the index, it would be used for 1) Home page, loading all the pins 2) the user's pin

Will change that in the rails controller when we set up users info

In short, yes we're loading all the pins... for now